src/battlearena/SearchPlayers.js

```diff
@@ -0,0 +1,93 @@
+//@flow strict
+
+import * as React from 'react';
+
+type Props = {|
+	playerList: Array<string>,
+|};
+
+type State = {|
+	searchName: string,
+	searchList: Array<string>
+|};
+
+// Search Players Component. Takes an array of all players.
+//
+// Prop Names:
+// playerList (Array of all users)
+//
+// State Names:
+// searchName (the keyword the user is searching with)
+// searchList (list of all users that match searchName)
+// 
+// EXAMPLE PROP USAGE = <SearchPlayers playerList={ ["jeffery", "jeff", "john", "illeatbutt", "childeatbigtime", "childconsumer"] }/>
+//
+// Needs to be updated to have a challenge player button.
+class SearchPlayers extends React.Component<Props, State> {
+
+	constructor(props: Props) {
+		super(props);
+
+		this.state = {
+			searchName: "",
+			searchList: []
+		}
+	}
+
+	// When typing in the searchbar, update searchName.
+	handleChange = (e: string) => {

```

Comment:
Can I get a return type here please?
I think it looks like:
handleChange = (e: string): void => {
}
but I could be wrong, because I usually wrap my methods when I call them instead of when I hand them out as callbacks instead of when I define them.